src/DataPersister/ChainDataPersister.php

```diff
@@ -63,7 +63,9 @@ public function remove($data)
     {
         foreach ($this->persisters as $persister) {
             if ($persister->supports($data)) {
-                return $persister->remove($data);
+                $persister->remove($data);
+
+                return;
```

Comment:
The return type should be strict. We do not add a `void` return type for now as that'd be a BC break.